generators/react/generator.ts

```diff
@@ -118,6 +118,14 @@ export default class ReactGenerator extends BaseApplicationGenerator {
             }),
           );
         };
+        if (application.clientRootDir) {
+          // Overrides only works if added in root package.json
+          this.packageJson.merge({
+            overrides: {
+              'browser-sync': application.nodeDependencies['browser-sync'],
+            },
+          });
+        }
```

Comment:
Needs to be run in postWriting otherwise the modified package.json will be overridden in writing priority.